orderer/consensus/smartbft/signer.go

```diff
@@ -65,7 +65,7 @@ func (s *Signer) SignProposal(proposal types.Proposal, _ []byte) *types.Signatur
 	}
 }
 
-// NewSignatureHeader creates a SignatureHeader with the correct signing identity and a valid nonce
+// newIdentifierHeaderOrPanic creates a SignatureHeader with the correct signing identity and a valid nonce
```

Comment:
```suggestion
// newIdentifierHeaderOrPanic creates an IdentifierHeader with the signer's identifier and a valid nonce
```
The descriptions should also be modified to match the actual function behavior.

Reply:
Thanks!  Modified!